tools/ka-customizer-installer

```diff
@@ -3,6 +3,9 @@
 APPDIR=$HOME/.local/share/applications
 BINDIR=$HOME/.local/bin
 
+if ! flatpak info --show-commit com.endlessm.Sdk eos3.1; then
+    flatpak install eos-runtimes com.endlessm.Sdk eos3.1
+fi
```

Comment:
shouldn't we check for and include com.endlessm.Platform eos3.1 as well?

btw, we could also use a variable here ($BRANCH?) so we can easily replace eos3.1 later on (or even have a installer.in version that does it automatically)?